online-shop/src/app/modules/shared/shopping-cart/services/shopping-cart-service.ts

```diff
@@ -10,6 +10,7 @@ export class ShoppingCartService {
   constructor() {
     // Încărcăm produsele din Local Storage în momentul inițializării serviciului
     this.loadCartItems();
```

Comment:
Seems like method `loadCartItems()` is only called in the constructor. If you don't use it anywhere else I would suggest to move the code directly here. As an additional point, I would suggest to initialize data on ngOnInit [()](https://jhapriti09.medium.com/what-is-the-difference-between-constructor-and-ngoninit-839748fc59ce)

Reply:
Removed constructor, I now use the code from loadCartItems() on ngOnInit();